annotations/tests/content.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "../test-suite.schema.json",
+  "description": "The content vocabulary",
+  "suite": [
+    {
+      "description": "`contentMediaType` is an annotation",
+      "compatibility": "7",
```

Comment:
The test looks good, but annotations emission is only really defined for 2019-09 onwards. If so, should this be 2019-09? I'm not sure we should be testing that previous drafts emit annotations in the same way as 2019-09. At least my implementation doesn't.

Reply:
I had a note about this in the original issue. I should probably put it in the readme as well. Although the concept of annotations didn't appear until 2019-09, the concept is compatible with previous dialects. Some implementations (like mine) will allow people to annotate instances with older schemas. So, this allows implementers to run tests for older dialects if they choose to support it.

Reply:
Yeah, I guess that makes sense! I'm not against it.